examples/vpc/s3/README.md

```diff
@@ -0,0 +1,3 @@
+## examples/vpc/s3
```

Comment:
I think I'd prefer not to nest examples too deeply. What about just creating a new folder`examples/s3` and then just sticking both examples into the same `example.tf` file?

Reply:
👍, Will update.